plugins/modules/swim_intent.py

```diff
@@ -791,24 +805,25 @@ def get_want(self, config):
             elif want["import_type"] == "local":
                 want["local_import_details"] = config.get("import_image_details").get("local_image_details")
             else:
+                self.log("Incorrect import type. Supported Values: local or url", "CRITICAL")
```

Comment:
self.log("The import type '{0}' provided is incorrect. Only 'local' or 'url' are supported.", "CRITICAL")

Specify the type

Reply:
added.